microceph/ceph/bootstrap.go

```diff
@@ -30,12 +30,19 @@ func Bootstrap(s common.StateInterface) error {
 	// Generate a new FSID.
 	fsid := uuid.NewRandom().String()
 	conf := newCephConfig(pathConsts.ConfPath)
-	err := conf.WriteConfig(
+	pubNet, err := common.FindNetworkAddress(s.ClusterState().Address().Hostname())
+	if err != nil {
+		return nil
+	}
+
+	err = conf.WriteConfig(
 		map[string]any{
-			"fsid":     fsid,
-			"runDir":   pathConsts.RunPath,
+			"fsid":   fsid,
+			"runDir": pathConsts.RunPath,
+			// First monitor bootstrap IP as passed to microcluster.
```

Comment:
I wonder: why are we setting up the initial mon address with the default ipaddr, even if the user might have passed in a separate `--mon-ip`? It would be good to have some rationale for this approach here